TestAssets/TestProjects/ResourcesTests/TestApp/project.json

```diff
@@ -7,7 +7,7 @@
     "Microsoft.Data.OData": "5.6.4",
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.0.0-rc2-24008"
+      "version": "1.0.0-rc2-24015"
```

Comment:
Wait - this is wrong.  Microsoft.NETCore.App needs to have 1.0.0-rc2-* since it comes from the CLI repo.